src/main/main.c

```diff
@@ -234,7 +234,7 @@ void init(void)
     servoMixerInit(masterConfig.customServoMixer);
 #endif
 
-#ifdef SPEKTRUM_BIND
+#if defined(SERIAL_RX) && defined(SPEKTRUM_BIND)
```

Comment:
Ah, of course. Sorry. 
There is a dependency to include order, common.h is included early, any undefs later on does not have any effect on what is defined or not in common.h . Sorry, keep it as is then. Or for full completeness, use the same conditions as in common.h . Add USE_SERIAL_RX_SPEKTRUM also. This would cover the case of serial_rx, but without spektrum support. Unlikely case, but for completeness any coverage of the Murphy cases. "If it can fail, it will fail."

    #if defined(SERIAL_RX) && defined(USE_SERIALRX_SPEKTRUM) && defined(SPEKTRUM_BIND)

Too bad complexity like this has to be duplicated in several places.. but that is really not a problem for this PR.